src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -79,9 +76,7 @@ function CreatePollContainer({
       <TouchableOpacity style={S.polldurationbutton} onPress={() => setIsDurationModalShown(true)}>
         <View style={S.row}>
           <Text style={S.fillparenttext}>Duration</Text>
-          <Text style={[S.polldurationbuttontext, {backgroundColor: dynamicColor.primary}]}>
-            {getDurationTimeText(selectedtime)}
-          </Text>
+          <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the dynamic color from the `Text` component may affect the visual consistency of your app if you were using it to adapt the color scheme based on the anonymity status. If this was not intended, consider adding it back.
```diff
- <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+ <Text style={[S.polldurationbuttontext, {backgroundColor: dynamicColor.primary}]}>{getDurationTimeText(selectedtime)}</Text>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->